src/actions/versionInfo.js

```diff
@@ -15,7 +16,7 @@ export const getVersionInfo = locale =>
 	makeOrcApiAction(
 		GET_VERSION_INFO,
 		buildUrl(["diagnostic", "versioninfo"], {
-			cultureName: locale,
+			cultureName: locale.substr(0, 2).toLowerCase() === "fr" ? locale : cultureByDefault,
```

Comment:
If it is Italian, we will default to English.... I guess this is Ok for now, but Italian will be translated eventually.

Reply:
The decision was from Product Owner.